src/messages/MOSDMap.h

```diff
@@ -30,7 +30,8 @@ class MOSDMap final : public Message {
   uint64_t encode_features = 0;
   std::map<epoch_t, ceph::buffer::list> maps;
   std::map<epoch_t, ceph::buffer::list> incremental_maps;
-  epoch_t oldest_map =0, newest_map = 0;
+  epoch_t cluster_osdmap_trim_lower_bound = 0; // formerly oldest_map
```

Comment:
```
  /**
   * cluster_osdmap_trim_lower_bound
   *
   * Encodes a lower bound on the monitor's osdmap trim bound.  Recipients
   * can safely trim up to this bound.  The sender stores maps back to
   * cluster_osdmap_trim_lower_bound.
   * 
   * This field was formerly named oldest_map and encoded the oldest map
   * stored by the sender.  The primary usage of this field, however, was to
   * allow the recipient to trim.  The secondary usage was to inform the
   * recipient of how many maps the sender stored in case it needed to request
   * more.  For both purposes, it should be safe for an older OSD to interpret
   * this field as oldest_map, and it should be safe for a new osd to interpret
   * the oldest_map field sent by an older osd as
   * cluster_osdmap_trim_lower_bound.  See bug <insert_tracker_url_here>
   */